app/models/manageiq/providers/ibm_cloud/power_virtual_servers/storage_manager/cloud_volume.rb

```diff
@@ -7,6 +7,73 @@ def available_vms
     availability_zone.vms
   end
 
+  def self.params_for_create()
+    {
+      :fields => [
+        {
+          :component => 'switch',
+          :name      => 'shareable',
+          :id        => 'shareable',
+          :label     => _('Shareable'),
+          :onText    => _('Yes'),
+          :offText   => _('No')
+        },
+        {
+          :component    => 'select',
+          :name         => 'affinity_policy',
+          :id           => 'affinity_policy',
+          :label        => _('Affinity Policy'),
+          :initialValue => 'off',
+          :options      => [
+            {
+              :label => 'Off',
+              :value => 'off'
+            },
+            {
+              :label => 'Affinity',
+              :value => 'affinity'
+            },
+            {
+              :label => 'Anti-affinity',
+              :value => 'anti-affinity'
+            }
+          ]
+        },
+        {
+          :component => 'select',
+          :name      => 'volume_type',
+          :id        => 'volume_type',
+          :label     => _('Cloud Volume Type'),
+          :options   => self.class.CLOUD_VOLUME_TYPES, # TODO: the types are in the angular controller, need to extract them
```

Comment:
@agrare @jaywcarman I am not aware of the cloud volume types for this provider, does it have any? [This](https://github.com/ManageIQ/manageiq-providers-amazon/pull/657/files#diff-8f332bdbb8551a6a7e20fb13ad23aa04R5-R11) is what I have for amazon.

Reply:
We're using the `cloud_volume_types` table. [OpenStack does the same](https://github.com/ManageIQ/manageiq-providers-openstack/blob/master/app/models/manageiq/providers/openstack/inventory/parser/storage_manager/cinder_manager.rb#L65).

Reply:
As far as I saw, the OpenStack form doesn't expose the `volume_type` field in its form.

Reply:
There is a "Cloud Volume Type" field for cinder volumes, and I think the volumes come from the `cloud_volume_types` table:

https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/views/cloud_volume/_common_new_edit.html.haml#L81-L92
![image](https://user-images.githubusercontent.com/1637291/95744047-5ce37d00-0c58-11eb-8157-abce7a8aefca.png)

Reply:
Okay, my bad, I see it has a `CloudVolumeType` model, will update this accordingly.